plugin/src/main/java/uk/co/deliverymind/wiremock/maven/plugin/WireMockMojo.java

```diff
@@ -2,17 +2,47 @@
 
 import com.github.tomakehurst.wiremock.standalone.WireMockServerRunner;
 import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugin.descriptor.PluginDescriptor;
 import org.apache.maven.plugins.annotations.LifecyclePhase;
 import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.apache.maven.plugins.annotations.ResolutionScope;
+import org.codehaus.plexus.classworlds.realm.ClassRealm;
 
+import java.io.File;
+import java.net.MalformedURLException;
 import java.util.Arrays;
+import java.util.List;
 
-@Mojo(name = "run", defaultPhase = LifecyclePhase.PRE_INTEGRATION_TEST)
+@Mojo(name = "run", defaultPhase = LifecyclePhase.PRE_INTEGRATION_TEST, requiresDependencyResolution = ResolutionScope.TEST)
```

Comment:
With requiresDependencyResolution set to TEST, lots of things get added to the classpath: guava, junit, jetty... I don't see the reason why this would be necessary. In my eyes ResolutionScope.RUNTIME would suffice.